### ISSUE TITLE
Opening app after restoring config throws back to onboarding screen

### ISSUE THREAD

**Issue by Username1-a** [Mahmud0808/ColorBlendr#74]
The issue is that when restoring theme config in Colorblendr, for the color changes to take effect, the app needs to be cleared from recents and relaunched again and with that it shows the setup screen (Permissions, Root/Shizuku) in which you need to pick Root or Shizuku, after which you need to tick some toggle/change something in-app and only then the new coloring will apply. It needs to be done each time you restore the theme config

(This setup screen)
![Screenshot_20240419-185630_Lawnchair](https://github.com/Mahmud0808/ColorBlendr/assets/47947180/50922f71-2543-4a28-8e1c-bcb51f4699db)

Basically, that it would be great if process of restoring theme configs could've been more seamless/streamlined

This issue persisted also on pre v1.4 versions of Colorblendr, so sorry for mentioning it that late

I'm using Android 13 QPR3, EvoX Custom ROM
